wm/border.go

```diff
@@ -53,20 +53,42 @@ func NewBorder(win fynedesk.Window, icon fyne.Resource, canMaximize bool) *Borde
 	}}
 
 	title := canvas.NewText(win.Properties().Title(), theme.ForegroundColor())
-	titleBar := newColoredHBox(win.Focused(), win, makeFiller(0),
-		newCloseButton(win),
-		max,
-		min,
-		title,
-		layout.NewSpacer())
+	buttonPos := fyne.CurrentApp().Preferences().StringWithFallback("toolbarbuttonposition", "Left")
```

Comment:
I could not really figure out how to get this working using the settings code. Any advice would be appreciated.

Reply:
This looks OK - what part of it isn't working as you'd like?

Reply:
I just thought that I'd be able to use the actually DeskSettings struct, but if this is the good way to go, I'm happy with that.